app/main.py

```diff
@@ -2,0 +1,17 @@
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+    mv, source_file, destination_file = split_command
+    path = destination_file[0:destination_file.rfind("/")]
+    if mv == "mv":
+        if "/" in command:
```

Comment:
Code Efficiency Issue: The check for `mv` command is unnecessary. It's already clear from the function name `move_file` that we are dealing with a `mv` command. You can safely remove it.